### DISCUSSION TITLE
Add Google Drive support

### DISCUSSION THREAD

**Discussion by SomeBasicUser** [UltimaHoarder/UltimaScraper#1357] [Ideas]
Make an option to transfer the downloaded content directly to Google Drive

**Comment by UltimaHoarder** [comment]
In my custom script, I use rclone to upload as soon as something has finished downloading in the background

**Reply by yokonjuan** [reply]
> 
> 
> In my custom script, I use rclone to upload as soon as something has finished downloading in the background

could you make this public? I use rclone to upload to g-drive but I do it manually... lol
